src/services/StorageService.ts

```diff
@@ -0,0 +1,97 @@
+import AWS from 'aws-sdk';
+
+const s3 = new AWS.S3({ region: process.env.BUCKET_REGION });
+export class StorageService {
+  async uploadFile(
+    fileName: string,
+    file: Express.Multer.File,
+    options?: Object
+  ): Promise<string | null> {
+    let fileNameKey = fileName;
+    if (options) {
+      if ('appendFileName' in options) {
+        fileNameKey = `${fileName}_${options['appendFileName']}`;
+      }
+    }
+    const params = {
+      Bucket: process.env.BUCKET_NAME,

```

Comment:
as mentioned above we should try to keep process.env only in config.ts; will update documentation for this sorry!

Reply:
done